src/components/__tests__/Index.js

```diff
@@ -99,3 +114,80 @@ it('renders correctly (with slot used)', () => {
 
   expect(wrapper.html()).toMatchSnapshot();
 });
+
+it('Allows a change in `index-name`', () => {
+  const wrapper = mount(Index, {
+    propsData: {
+      searchClient: {},
+      indexName: 'bla',
+    },
+  });
+
+  wrapper.setProps({
+    indexName: 'doggie_bowl',
+  });
+
+  expect(instantsearch.__helper.setIndex).toHaveBeenCalledTimes(1);
```

Comment:
Wouldn't it make more sense to expect `instantsearch.__helper.getIndex()` to be `doggie_bowl` instead of the two `setIndex` lines that are more testing the helper implementation instead? ([see reference](https://community.algolia.com/algoliasearch-helper-js/reference.html#AlgoliaSearchHelper#getIndex))

Reply:
yeah sure, that's possible :)